docs/index.md

```diff
@@ -101,12 +101,17 @@ As [measured in April 2020](https://github.com/kreier/solarmeter/blob/master/REA
 
 Further measurement has been conducted in November 2020. The results.
 
+- Running 66 mA
+- WiFi 80-140 mA
+- CPU 80 MHz 48 mA
+- Light Sleep 9 mA
+
```

Comment:
How much deep sleep mA?

Reply:
The tested ones still had something in the mA range. Need to check the different boards.